### PR TITLE
add yup tokens

### PR DESCRIPTION
**Pull request by darren-wangg**
## How to add token?

- [x] Add metadata to [`tokens.json`](tokens.json)

```json
{
    "name": "<NAME>",
    "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/<SYMBOL>.png",
    "symbol": "<SYMBOL>",
    "account": "<CONTRACT NAME>",
    "chain": "eos"
}
```

- [x] Add token logo `*.png` to [`./logos`](./logos) folder
- [x] Update `README.md` with **Symbol** & **Contract Name**

### CODE REVIEW

**Comment by darren-wangg** [comment]
@jafri @DenisCarriere does this look good? Please let me know if I'm missing anything. Thanks

File: tokens.json [hunk 1]
```diff
@@ -1846,13 +1846,29 @@
...
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUP",
+        "account": "token.yup",
+        "chain": "eos"
+    },
+    {
+        "name": "YUPETH LP",
+        "logo": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "logo_lg": "https://raw.githubusercontent.com/eoscafe/eos-airdrops/master/logos/YUP.png",
+        "symbol": "YUPETH LP",
```
Comment (DenisCarriere) [inline tokens.json:1868]: The symbol code seems wrong:
1. There's a white space
2. 8 character symbol codes is invalid

File: tokens.json [hunk 2]
```diff
@@ -1846,13 +1846,29 @@
...
+        "account": "lptoken.yup",
```
Comment (DenisCarriere) [inline tokens.json:1869]: token contract is not deployed to this account

**Review by DenisCarriere — CHANGES_REQUESTED** [review]
YUPETH LP token is invalid

File: tokens.json (hunk 2 above)
Comment (darren-wangg) [reply tokens.json:1869]: token contract will be ready by end of day

Comment (Velua) [reply tokens.json:1869]: We can't accept tokens that don't yet exist, let us know when it does.

Comment (darren-wangg) [reply tokens.json]: got it, no problem. I submitted a PR without the lp token for now, could that be approved for now?

Comment (DenisCarriere) [reply tokens.json]: Also your token symbol is invalid, it can’t have 8 characters, the maximum limit is 7 characters

Comment (darren-wangg) [reply tokens.json]: @DenisCarriere got it, thank you. I just removed the yupeth lp token entry entirely. Is this looking good now?

**Comment by pinkhoodie** [comment]
@DenisCarriere can we please update this? Thanks

**Pull request merged**